OneginiExampleAppSwift/Interactors/RegisterUserInteractor.swift

```diff
@@ -44,7 +44,7 @@ extension RegisterUserInteractor: RegisterUserInteractorProtocol {
     }
 
     func startUserRegistration(identityProvider: ONGIdentityProvider? = nil) {
-        ONGUserClient.sharedInstance().registerUser(with: identityProvider, scopes: ["read", "write"], delegate: self)
+        ONGUserClient.sharedInstance().registerUser(with: identityProvider, scopes: ["read", "openid"], delegate: self)
```

Comment:
Can we have an enum with all possible options?

Reply:
that's just an example of usage, one can change it to whatever he wants if implemented in the destination app

Reply:
Hmm I wish we had SDK in Swift :)